blockdevice/blockdevice.go

```diff
@@ -282,6 +306,12 @@ type TemperatureInformation struct {
 
 	// CurrentTemperature is the temperature of the drive in celsius
 	CurrentTemperature int16
+
+	// LowestTemperature is the lowest temperature of the drive recorded in celsius
+	LowestTemperature int16
+
+	// HighestTemperature is the highest temperature of the drive recorded in celsius
+	HighestTemperature int16
```

Comment:
Seachest also reports whether the reported highest/lowest temperature value is valid or not. Those fields also should be included as part of the metrics.

Reply:
Will add that. A bit confused about what does validity means here though. Is it whether that's applicable to the block device or something else?

Reply:
To the blockdevice, every temperate metric that seachest gives also has an associated field which tells whether the reported data is valid or not. We have already added that field to the current temperature metric. It need to be added to the highest and lowest temperature also.

Reply:
@harshthakur9030  PTAL.

Reply:
Added lowest and highest temperature valid.